R/zzz.r

```diff
@@ -10,5 +10,12 @@
   makeActiveBinding('run', function() run_model, .GlobalEnv)
 
   addTaskCallback(auto_run)
-}
 
+  tryCatch(syberia_project(), error = function(e) {
```

Comment:
If not just write a utility function `is.syberia_project` that takes a path? `tryCatch`ing is a terrible idea here.